### PR TITLE
Voting and commenting

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @btholt or @jo-asakura or @danehansen

### CODE REVIEW

File: src/views/components/CommentBox.jsx [hunk 1]
```diff
@@ -54,10 +57,10 @@ class CommentBox extends React.Component {
     return (
       <div className='row CommentBox'>
         <div className='col-xs-12'>
-          <form action={ '/comment' } method='POST' onSubmit={ this.submit }>
+          <form action={ '/comment' } method='POST' onSubmit={ this.submit.bind(this) }>
             <div className='form-group'>
               <label className='sr-only' htmlFor={ 'textarea-' + this.props.thingId }>Comment</label>
-              <textarea placeholder='Add your comment!' id={ 'textarea-' + this.props.thingId } rows='2' className='form-control' name='text'></textarea>
+              <textarea placeholder='Add your comment!' id={ 'textarea-' + this.props.thingId } rows='2' className='form-control' name='text' ref='text'></textarea>
```
Comment (btholt) [inline src/views/components/CommentBox.jsx]: Is this `ref` used anywhere?

**Comment by btholt** [comment]
:nail_care: If that `ref` is actually important, then go ahead and leave it and consider this fish'd. If it isn't, then just take it out and then I'm good with it. :smile:

File: src/views/components/CommentBox.jsx (hunk 1 above)
Comment (ajacksified) [inline src/views/components/CommentBox.jsx]: https://github.com/reddit/reddit-mobile/pull/52#discussion-diff-27683492R13

**Comment by ajacksified** [comment]
It is - that's how it pulls out the text.

**Comment by btholt** [comment]
:fish: Cool.

**Pull request merged**